platform/broadcom/docker-syncd-brcm/start.sh

```diff
@@ -10,6 +10,8 @@ supervisorctl start syncd
 
 # If this platform has an initialization file for the Broadcom LED microprocessor, load it
 if [ -r ${PLATFORM_DIR}/led_proc_init.soc ]; then
+    # Sleep a bit to allow syncd to create the socket for bcmcmd to connect to
+    sleep 5
```

Comment:
Wait until the socket is active
Like this
while /bin/true;
do

 if [[ -e /var/run/sswsyncd/sswsyncd.socket ]];
 then
   break
 fi
 echo sleep
 sleep 1

done